.github/workflows/build-website.yml

```
@@ -51,16 +51,15 @@ jobs:
           ./ENV/bin/python ./tools/check-links http://localhost:9001/
           kill %1
         shell: bash
-      - name: check links (build)
+      - name: check links (apache2)
         run: |
-          cd website/www/
-          python3.7 -m http.server 9002 &
-          sleep 0.5  # HACK(strager): Wait for server to start.
-          cd ../../
-          ./ENV/bin/python ./tools/check-links http://localhost:9002/
-          kill %1
+          sudo apt-get install apache2 -y
```

Nit: Include `DEBIAN_FRONTEND=noninteractive`, just in case:
```suggestion
          DEBIAN_FRONTEND=noninteractive sudo apt-get install apache2 -y
```

---

I'm not sure if it's an issue with your `check-links` script or something I'm doing, but your script is not failing the workflow when there's a syntax error in the `.htaccess` file.

 See https://github.com/tonysathre/quick-lint-js/runs/4645869307?check_suite_focus=true#step:14:236